<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DzMarkEditer</class>
 <widget class="QWidget" name="DzMarkEditer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>551</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>989</width>
      <height>549</height>
     </rect>
    </property>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="Line" name="titleHorLine">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>118</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="titleLbl">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLbl">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="excelButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="noteTextEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>321</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="verdictBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>42</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="maxMarkLbl">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="fioLbl">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
